bioconda_utils/bioconda_utils-conda_build_config.yaml

```diff
@@ -21,44 +21,35 @@ pin_run_as_build:
     max_pin: x.x
     min_pin: x.x
 
-# Update GLPK ahead of general pinnings for the Bioconductor 3.16 bulk update.
-# TODO: Remove this once we update to conda-forge-pinning>=2022.10.04.12.34.07.
-glpk:
-  - 5
-
 htslib:
-  - 1.17
+  - "1.17"
 
 bamtools:
-  - 2.5.1
+  - "2.5.1"
 
 # NOTE: Workaround https://github.com/conda/conda-build/issues/3974 we slightly alter the values
 #       from conda-forge-pinnings here (inserting '.*' or ' ' which should be ignored later on).
 r_base:
-  - 4.2.*
+  - "4.2.*"
 
 python:
   - 3.10.*  *_cpython
   - 3.9.*  *_cpython
   - 3.8.*  *_cpython
-  - 3.7.*  *_cpython
-  - 3.6.*  *_cpython
   - 2.7.*  *_cpython
```

Comment:
It's their a reason to keep python2? People could add local pinning files if they really need to.

Reply:
Good point. You are completely right I think. I will remove it.